src/components/subheader/components/discountSection/DiscountSection.js

```diff
@@ -0,0 +1,33 @@
+// Library
+import PropTypes from 'prop-types'
+
+// component
+import OffersList from '../offersList'
+
+// css
+import './discountSection.css'
+
+
+function DiscountSection(props){
```

Comment:
Call it as OfferSection because from this component you're rendering OfferList, so they've similar meanings so you should be consistent with this